name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/SantanderConsumerBankPDFExtractor.java

```diff
@@ -185,4 +192,98 @@ private <T extends Transaction<?>> void addFeesSectionsTransaction(T transaction
                 .match("^Fremde Abwicklungsgeb.hr .* (?<fee>[.,\\d]+)\\- (?<currency>[\\w]{3})$")
                 .assign((t, v) -> processFeeEntries(t, v, type));
     }
+
```

Comment:
please shift the function about the treatment of taxes and fees (to line 168...)

Structure: [Contruting rules](https://github.com/buchen/portfolio/blob/master/CONTRIBUTING.md#anatomy-of-a-pdf-importer)

- Client + Importer label
- Transaction types
- Taxes and fees 
- other helping classes

All other importers are set up like this.
Simple task for you... and quickly done. 👍🏻